nemo/collections/llm/gpt/model/base.py

```diff
@@ -326,7 +326,7 @@ def configure_model(self, tokenizer, pre_process=None, post_process=None, vp_sta
         Returns:
             MCoreGPTModel: Configured Megatron Core GPT model instance
         """
-        if self.enable_cuda_graph:
+        if self.enable_cuda_graph and self.cuda_graph_scope != "full_iteration":
```

Comment:
Please a comment of `Enable per-Transformer layer cuda graph` above

Reply:
Done